numba_cuda/numba/cuda/cuda_paths.py

```diff
@@ -25,15 +45,17 @@ def _find_valid_path(options):
 
 
 def _get_libdevice_path_decision():
-    options = [
-        ("Conda environment", get_conda_ctk()),
-        ("Conda environment (NVIDIA package)", get_nvidia_libdevice_ctk()),
-        ("CUDA_HOME", get_cuda_home("nvvm", "libdevice")),
-        ("System", get_system_ctk("nvvm", "libdevice")),
-        ("Debian package", get_debian_pkg_libdevice()),
-    ]
-    by, libdir = _find_valid_path(options)
-    return by, libdir
+    options = _build_options(
+        [
+            ("Conda environment", get_conda_ctk()),
+            ("Conda environment (NVIDIA package)", get_nvidia_libdevice_ctk()),
+            ("CUDA_HOME", get_cuda_home("nvvm", "libdevice")),
+            ("NVIDIA NVCC Wheel", get_libdevice_wheel()),
+            ("System", get_system_ctk("nvvm", "libdevice")),
+            ("Debian package", get_debian_pkg_libdevice()),
+        ]
```

Comment:
I know it was already done this way, but it would be nice to not have to resolve every possible path if we can short circuit.